internal/controller/postgrescluster/pgmonitor.go

```diff
@@ -401,10 +365,17 @@ func configureExporterTLS(cluster *v1beta1.PostgresCluster, template *corev1.Pod
 		}}
 
 		exporterContainer.VolumeMounts = append(exporterContainer.VolumeMounts, mounts...)
-		exporterContainer.Command = pgmonitor.ExporterStartCommand([]string{
-			pgmonitor.ExporterExtendQueryPathFlag, pgmonitor.ExporterWebListenAddressFlag, pgmonitor.ExporterWebConfigFileFlag,
-		})
+		exporterContainer.Command = pgmonitor.ExporterStartCommand(
+			withBuiltInCollectors, pgmonitor.ExporterWebConfigFileFlag)
 	}
+
+	template.Spec.Containers = append(template.Spec.Containers, exporterContainer)
+
+	// add the proper label to support Pod discovery by Prometheus per pgMonitor configuration
+	initialize.Labels(template)
+	template.Labels[naming.LabelPGMonitorDiscovery] = "true"
```

Comment:
OK, all the shuffling / simplification here makes sense; but this part seems new. Why do we need this now when we didn't before?

Reply:
This was on L347, after `configureExporterTLS`. It's in the same spot but looks new in the diff.